### ISSUE TITLE
Is this the code for the CVPR2022 paper?

### ISSUE THREAD

**Issue by hello-world-hhh** [hanna-xu/MURF#2]
Hello, thank you for the code! 
But is this the code for the CVPR2022 work "RFNet: Unsupervised Network for Mutually Reinforcing Multi-modal Image Registration and Fusion"? I notice there are several differences between this code and the paper (for example, the code uses contrast learning instead of the modal transfer net in the paper). Or is this just a code for another work? 
Sincerely looking forward to your reply, thank you!

**Comment by hanna-xu** [comment]
Thanks for this question. This is a code for another work.

**Comment by hello-world-hhh** [comment]
Thank you for your reply! 
I have a question about the training of **TransNet**(modal transfer) in your CVPR2022 paper (RFNet). I would like to ask whether the paired VIS and NIR images used for training are already registered? If they are not pre-registered, would it be inappropriate to calculate the L1 similarity between the transformed VIS image and the NIR image in the loss function? (Because this similarity measurement is pixel-wise, but the pixels between two images with positional deviation are not one-to-one correspondence.)
Looking forward to your reply, thank you!

**Comment by yidamyth** [comment]
The articles from CVPR 2022 seem to lack code. According to the structural diagram of the paper, the first part is coarse registration, so it consists of unregistered data pairs.  In my view, the use of L1 loss is merely to constrain the transformed image to maintain consistency with the original image as much as possible. To make the two images closer, there is no strict one-to-one correspondence requirement since it's coarse registration. I am also a reader of this paper, and if you have time, perhaps we could exchange ideas and learn from each other.

**Comment by xuliuwei** [comment]
@YidaMyth 请问您方便加个微信吗？

**Comment by Kevinskr** [comment]
@yidamyth 请问这个仓库的代码您跑通了吗，我跑了一下loss一直是nan，现在也没有调好

**Comment by yidamyth** [comment]
> @yidamyth 请问这个仓库的代码您跑通了吗，我跑了一下loss一直是nan，现在也没有调好

您好，训练的时候也遇见这个问题，所以直接跑的测试。
